.github/workflows/CI.yml

```diff
@@ -493,50 +451,74 @@ jobs:
             VERSION
           key: e2etest-release-${{ github.run_id }}
 
+      - name: restore cached rpms
+        uses: actions/cache@v2
+        with:
+          key: "cached_packages_${{ github.run_id }}"
+          path: $PACKAGING_ROOT
+
+      - name: Restore cached binaries
+        if: steps.e2etest-release.outputs.cache-hit != 'true'
+        uses: actions/cache@v2
+        with:
+          key: "cached_binaries_${{ github.run_id }}"
+          path: build
+
+      - name: Set up QEMU
```

Comment:
why does QEMU need to be setup

Reply:
It is supposed to support multiple OS, mostly the same as buildx but currently we don't need it so we can remote it.